components/VM_Arm/CMakeLists.txt

```diff
@@ -127,6 +127,20 @@ config_option(
     OFF
 )
 
+config_option(
+    VmCustomDtbBase
```

Comment:
better use `VmCustomDtbBaseFile` and `VM_CUSTOM_DTB_BASE_FILE` to make it clear this is a file name.

Reply:
Done.